shoop/core/models/_contacts.py

```diff
@@ -234,12 +239,18 @@ class Meta:
         verbose_name_plural = _('persons')
 
     def save(self, *args, **kwargs):
+        if self.first_name or self.last_name:
+            self.name = (" ".join([self.first_name, self.last_name])).strip()
```

Comment:
OK, we probably shouldn't add any new features right now, so let's not add middle names. If we want that later, we can do it then. Until that it's still possible to set `person.first_name = 'John Fitzgerald "Jack"'` and `person.last_name = 'Kennedy'` to make full names with more than two names.

I would still prefer that form for setting the names even though stripping is cool it doesn't belong to model level.